sormas-backend/src/test/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjbTest.java

```diff
@@ -551,6 +552,38 @@ public void searchEventParticipantsByPersonOtherDetail() {
 		eventParticipantIndexDtos = getEventParticipantFacade().getIndexList(eventParticipantCriteria, 0, 100, null);
 		assertEquals(0, eventParticipantIndexDtos.size());
 	}
+	@Test
+	public void testCreationOfEventParticipantWhenArchived()
+	{
+		TestDataCreator.RDCF rdcf = creator.createRDCF("Region", "District", "Community", "Facility");
+		UserDto user = createUser(rdcf);
+		EventDto event = createEvent(user, rdcf);
+		getEventFacade().archive(Collections.singletonList(event.getUuid()));
+		PersonDto eventPersonArchived = creator.createPerson("EventArchived", "OrganizerArchived");
+		EventParticipantDto archivedEventParticipantDto = creator.createEventParticipant(event.toReference(), eventPersonArchived, "event Director", user.toReference());
+		assertTrue(getEventParticipantFacade().isArchived(archivedEventParticipantDto.getUuid()));
+
+		getEventFacade().dearchive(Collections.singletonList(event.getUuid()), "reason");
+		PersonDto eventPersonActive = creator.createPerson("EventActive", "OrganizerActive");
+		EventParticipantDto activeEventParticipantDto = creator.createEventParticipant(event.toReference(), eventPersonActive, "event Director", user.toReference());
+		assertFalse(getEventParticipantFacade().isArchived(activeEventParticipantDto.getUuid()));
+	}
+
+	@Test
+	public void testCreationOfExistingEventParticipantRemainsArchived()
```

Comment:
This test is almost identical with the first test case. Please join both test cases (saves several seconds for unit test setup), and add code comments before each block to explain what that test step is testing. Example:

```java
// Verify that dearchiving of the event does not dearchive the event participant
```